src/test/java/com/easypost/ThreadTest.java

```diff
@@ -2,6 +2,8 @@
 
 import com.easypost.exception.EasyPostException;
 import com.easypost.model.Order;
+import com.easypost.service.EasyPostClient;
```

Comment:
I'm not seeing a thread test that spins up multiple threads with various API keys and gets the API keys out of those client objects to ensure they didn't get mixed up, can we please add that here?

Reply:
This is the old unit test, there is a test to ensure API keys do not get mixed up between multiple client objects in `EasyPostTest.java`.

Reply:
If so, the thread test should be placed in the thread test test file, don't you agree?

Reply:
sure